dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.0.json

```diff
@@ -583,14 +578,14 @@
       },
       {
        "value": "lms_integration",
-       "text": "Lack of integration with my school’s LMS"
+       "text": "Lack of integration with my school’s technology or LMS (Learning Management System)"
       }
      ]
     },
     {
      "type": "comment",
      "name": "one_challenge_FR_normal",
-     "title": "For one of the items you identified as a huge challenge, please describe any ideas you have for how code.org might address it.",
+     "title": "For one of the items you identified as a challenge, please describe any ideas you have for how code.org might address it.",
      "cols": 40,
      "rows": 6
```

Comment:
on the answers from lines 625-642, half the answers have `?` at the end while the others have periods. Is that intentional?

Reply:
good catch